server/graphql/v2/mutation/MemberMutations.ts

```diff
@@ -31,7 +32,88 @@ const isLastAdmin = async (account, memberAccount) => {
   return admins.length === 1 && admins[0].MemberCollectiveId === memberAccount.id;
 };
 
+const GraphQLFollowResult = new GraphQLObjectType({
+  name: 'FollowResult',
+  fields: {
+    individual: {
+      type: new GraphQLNonNull(GraphQLIndividual),
+    },
+    member: {
+      type: new GraphQLNonNull(GraphQLMember),
+    },
+  },
+});
+
+const GraphQLUnfollowResult = new GraphQLObjectType({
+  name: 'UnfollowResult',
+  fields: {
+    individual: {
+      type: new GraphQLNonNull(GraphQLIndividual),
+    },
+    member: {
+      type: GraphQLMember,
+    },
+  },
+});
+
 const memberMutations = {
+  follow: {
```

Comment:
I'd recommend calling these mutations `followAccount`/`unfollowAccount` for clarity; we already have a `followConversation` mutation and it's possible that we'll add more like that in the future (e.g. `followExpense`, `followThread`, etc).